### PR TITLE
Specify branch main on production_build.yaml and unit_tests.yaml

### PR DESCRIPTION
**Pull request by samsonasik**
To avoid production build and Unit test gha hanging:

Screen Shot 2021-06-23 at 12 40 12

### CODE REVIEW

**Comment by samsonasik** [comment]
It is strange, the "Cache warmup" and "unit_tests" keep showing "Waiting for status to be reported"

**Comment by TomasVotruba** [comment]
That's because those are "required" before merge.

Unit test must be invoked too:
https://github.com/rectorphp/getrector.org/blob/main/.github/workflows/unit_tests.yaml

**Comment by samsonasik** [comment]
How to disable them as they only on push, not pull request ?

**Comment by TomasVotruba** [comment]
> How to disable them as they only on push, not pull request ?

I would enable them on push too. Having passed unit test is crutial.

I've added this repository to your access. You should be able to merge now

**Comment by samsonasik** [comment]
They were only on push, do you mean they need to be run in "pull_requests" as well? For unit tests, that's ok, for "Cache warmup", that seems change production docker?

**Comment by samsonasik** [comment]
I enabled "Unit tests" for running on both "pull request" and "push", @TomasVotruba could you disable "Cache warmup" action from required in pull request, as it seems push to production?

**Comment by TomasVotruba** [comment]
Cache warmup must pass too. If it fails, the production might be pushed and broken (this happened yestereday due to Symfony renamed class BC break)

**Comment by samsonasik** [comment]
Ok, I enable "Cache warmup" to run on pull request too.

File: .github/workflows/production_build.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: Production build
```
Comment (TomasVotruba) [inline .github/workflows/production_build.yaml:1]: @samsonasik It seems I mixed the naming here. Could you update the file name + name to "cache_warmup"? It's definitely not a production build :)

File: .github/workflows/production_build.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 
-on: push
+on:
+    pull_request: null
+    push:
+        branches:
+            - main
```
Comment (TomasVotruba) [inline .github/workflows/production_build.yaml:7]: :+1:

File: .github/workflows/production_build.yaml (hunk 1 above)
Comment (samsonasik) [reply .github/workflows/production_build.yaml:1]: implemented https://github.com/rectorphp/getrector.org/pull/388

Comment (TomasVotruba) [reply .github/workflows/production_build.yaml:1]: Thanks! :slightly_smiling_face:  Now the confusion makes sense, I've picked very bad naming :/

**Pull request merged**